doc: minor CONTRIBUTING tweaks after onboarding

akavel opened the pull request:
[ci no changelog needed]

### Pull Request Description

<!--
- Please describe the nature of your PR here, as well as the motivation for it.
- If it fixes an open issue, please mention that issue number here.
-->

Small improvements I found missing while onboarding.

Also, based on info from @MichaelMauderer, rustfmt is now used in ide
codebase.



### Important Notes

<!--
- Mention important elements of the design.
- Mention any notable changes to APIs.
-->


### Checklist

Please include the following checklist in your PR:

- [ ] The documentation has been updated if necessary.
- [ ] All code conforms to the [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md), [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md), and [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guides.
- [ ] All documentation and configuration conforms to the [markdown](https://github.com/enso-org/enso/blob/develop/docs/style-guide/markdown.md) and [YAML](https://github.com/enso-org/enso/blob/develop/docs/style-guide/yaml.md) style guides.
- [ ] All code has been tested where possible.

akavel committed:
doc: minor CONTRIBUTING tweaks after onboarding

Small improvements I found missing while onboarding.

Also, based on info from @MichaelMauderer, rustfmt is now used in ide
codebase.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/enso-org/enso?pullRequest=3178) <br/>All committers have signed the CLA.

akavel committed:
Merge branch 'develop' into wip/akavel/onboarding-feedback-180508283

akavel committed:
doc: prettyprint

vitvakatu commented on app/gui/docs/CONTRIBUTING.md:
```diff
@@ -98,15 +105,13 @@ setup:
 
-We do, however, use [`prettier`](https://prettier.io/) for the JavaScript files
-in our code base. If you have not installed it already you can do so via
+We do also use [`prettier`](https://prettier.io/) for the JavaScript files in
```
```suggestion
We do also use [`prettier`](https://prettier.io/) for the JavaScript and Markdown files in
```

akavel committed:
trying to disable changelog assertion check

Empty commit with special annotation, attempting to disable changelog
assertion check for whole PR.

[ci no changelog needed]

akavel committed:
Update app/gui/docs/CONTRIBUTING.md

Co-authored-by: Ilya Bogdanov <fumlead@gmail.com>

akavel committed:
Clarify guide for PRs not needing changelog entry

Was mentioning commit message, but the check is actually working for PR
description too, and this approach is more practical.

akavel committed:
doc: prettyprint
